audio-worklet/src/bitcrusher-demo.js

```diff
@@ -124,13 +150,13 @@ class BitcrusherDemo {
    */
   switchBitcrusherNode() {
     // The function is called after the checkbox switches state.
-    const changeToScriptProcessor = !this.useAudioWorkletCheckBox_.checked;
-    if (changeToScriptProcessor) {
-      this.bitcrusherScriptProcessor_.output.connect(this.masterGain_);
-      this.bitcrusherAudioWorklet_.disconnect();
-    } else {
+    const changeToAudioWorklet = !this.useScriptProcessorCheckBox_.checked;
+    if (changeToAudioWorklet) {
       this.bitcrusherAudioWorklet_.connect(this.masterGain_);
       this.bitcrusherScriptProcessor_.output.disconnect();
+    } else {
+      this.bitcrusherScriptProcessor_.output.connect(this.masterGain_);
+      this.bitcrusherAudioWorklet_.disconnect();
```

Comment:
I thought we had discussed the switching would be done with a pair of gain nodes.  This connect/disconnect isn't sample-accurate.  Plus you connect before disconnect, so there's a chance you get output from both at the same time.

With gain nodes and AudioParam control, you can change them at the same time guaranteed (if you schedule a bit in the future).